.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "stable"
+# TODO: switch back to `npm test` default once tests are fixed.
+script: npm run lint
```

Comment:
Just follow the pattern I used here https://github.com/tastejs/todomvc/blob/master/.travis.yml